WPMedia/fusion-news-theme-blocks
# TMEDIA-645 - Remove unused amp code in oembed

## Description
Remove amp code in oembed 

## Jira Ticket
- [TMEDIA-645](https://arcpublishing.atlassian.net/browse/TMEDIA-645)

## Acceptance Criteria
No reference to `<amp-` functionality in the Oembed component

Non-amp embed html should still work

## Test Steps

1. Checkout this branch `git checkout REFACTOR-remove-unused-amp`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
3. Go to http://localhost/pf/test-article-page/?_website=the-gazette
4. Scroll down to see embeds like twitter, fb, etc render as before 

<img width="918" alt="Screen Shot 2022-01-27 at 14 18 14" src="https://user-images.githubusercontent.com/5950956/151443465-c1d51554-694b-4586-a7bd-4037e9440113.png">
<img width="863" alt="Screen Shot 2022-01-27 at 14 18 05" src="https://user-images.githubusercontent.com/5950956/151443513-7a937a95-e5e0-4f5f-bda8-66032ba77af9.png">

5. Notice how youtube is wide because of the news-theme-css class. This is due to deliberate logic 
<img width="890" alt="Screen Shot 2022-01-27 at 15 05 55" src="https://user-images.githubusercontent.com/5950956/151443550-b3ccb10f-87cb-4974-a41e-d0f4e02f6fd7.png">

## Effect Of Changes
### Before
- embeds that reference unsupported amp
- 
### After
- no change
- No reference to code that has amp in it

## Dependencies or Side Effects

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
